arelle/CntlrCmdLine.py

```diff
@@ -73,8 +75,8 @@ def parseArgs(args):
     and
     arellePluginModules which is a dictionary of commands and moduleInfos
     """
-
     uiLang = None
+    gettext.install("arelle")
```

Comment:
I think we've broken the translation of the command line help and error messages. Consider the uiLang evaluation logic just below this and the prior call to the [cntrl `setUiLanguage`](https://github.com/Arelle/Arelle/blob/2c522ed0ffe57e273d6596dd38bf62568d2eb4d8/arelle/Cntlr.py#L312-L314) from the cntrl's constructor